CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,134 @@
+
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+We as members, contributors, and leaders pledge to make participation in our
+community a harassment-free experience for everyone, regardless of age, body
+size, visible or invisible disability, ethnicity, sex characteristics, gender
+identity and expression, level of experience, education, socio-economic status,
+nationality, personal appearance, race, caste, color, religion, or sexual
+identity and orientation.
+
+We pledge to act and interact in ways that contribute to an open, welcoming,
+diverse, inclusive, and healthy community.
+
+## Our Standards
+
+Examples of behavior that contributes to a positive environment for our
+community include:
+
+* Demonstrating empathy and kindness toward other people
+* Being respectful of differing opinions, viewpoints, and experiences
+* Giving and gracefully accepting constructive feedback
+* Accepting responsibility and apologizing to those affected by our mistakes,
+  and learning from the experience
+* Focusing on what is best not just for us as individuals, but for the overall
+  community
+
+Examples of unacceptable behavior include:
+
```

Comment:
```suggestion
* Disrespectful behavior, mannerisms, or attacks
* Use of disrespectful, hostile, confrontational, or alienating language
```
I suggest adding a couple more bullet points at the beginning

Reply:
No objection to the second point (“use of disrespectful, … language“).

Unsure about the first. Contrary to the other examples given below, this one feels vague and too open to interpretation. What is an “attack”? What is a “disrespectful behavior”?

“You closed my issue without addressing it. That’s disrespectful!”

Reply:
Fair enough! Feel free to disregard these suggestions as you see fit. :) (I was looking at the OBO Foundry code of conduct for specific language that they had used - https://obofoundry.org/docs/COC.html, but I agree, this can be subjective.)